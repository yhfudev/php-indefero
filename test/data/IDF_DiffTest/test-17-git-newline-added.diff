diff --git a/foo b/foo
index 71ccdd0..a86c18f 100644
--- a/foo
+++ b/foo
@@ -1 +1 @@
-This is foo
\ No newline at end of file
+This is foo
