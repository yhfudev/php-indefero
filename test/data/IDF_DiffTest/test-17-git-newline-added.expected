<?php return array (
  'foo' => 
  array (
    'chunks' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => '',
          2 => "This is foo",
        ),
        1 => 
        array (
          0 => '',
          1 => '1',
          2 => "This is foo\n",
        ),
      ),
    ),
    'chunks_def' => 
    array (
      0 => 
      array (
        0 => 
        array (
          0 => '1',
          1 => 1,
        ),
        1 => 
        array (
          0 => '1',
          1 => 1,
        ),
      ),
    ),
  ),
);